.github/workflows/daily.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   audit:
     runs-on: ubuntu-latest
     container:
-      image: libra/build_environment:github-1
```

Comment:
hmm this is a bit of a problem here.   github-1 tag is no longer used in diem, I think switching the tag to "master" would let this keep functioning.

Reply:
Thanks for the heads up!  

Once #7871 landed, we would have the `main` image https://hub.docker.com/r/diem/build_environment/tags?page=1&ordering=last_updated.

Reply:
Will update.